libraries/AP_VideoTX/AP_VideoTX.h

```diff
@@ -158,6 +158,11 @@ class AP_VideoTX {
     bool get_enabled() const { return _enabled; }
     bool update_enabled() const { return _defaults_set && _enabled != _current_enabled; }
 
+    // set current vtx temperature (called by backend)
+    void set_temperature(uint16_t temperature) { _current_temperature = temperature; }
```

Comment:
you should record this in appropriate units and document, probably oC recorded as a float

Reply:
changed to float